src/plugins/header/options/options.ts

```diff
@@ -43,10 +49,24 @@ export class UIPOptions extends UIPPlugin {
     super.disconnectedCallback();
   }
 
-  protected render() {
-    const options = UIPOptions.UIPOptionsConfig.filter(option => !option.canActivate || option.canActivate(this));
-    const entries: [string, UIPOption][] = options.map(option => [option.attribute, UIPOption.create(option)]);
-    this.options = new Map(entries);
+  protected attributeChangedCallback(attrName: string, oldVal: string | null, newVal: string | null) {
+    if (oldVal === newVal || !this.options) return;
+
+    const config = UIPOptions.UIPOptionsConfig.find((elem) => elem.attrName === attrName);
```

Comment:
And here I would do like this:
```suggestion
    const config: OptionConfig | undefined = UIPOptions.UIPOptionsConfig.find(elem => elem.attrName === attrName);
```